common/jmeter/oauth/OAuth_Client_Credentials_Grant.jmx

```diff
@@ -229,7 +229,7 @@ vars.put(&quot;tokenEndpoint&quot;,new String(tokenEndpoint));
 </stringProp>
         </BeanShellPreProcessor>
         <hashTree/>
-        <HTTPSamplerProxy guiclass="HttpTestSampleGui" testclass="HTTPSamplerProxy" testname="Get Token Client Credential Grant" enabled="true">
+        <HTTPSamplerProxy guiclass="HttpTestSampleGui" testclass="HTTPSamplerProxy" testname="1 Get Token Client Credential Grant" enabled="true">
```

Comment:
Check the impact on the summary-modify.py file when preparing the summary.
Don't we need to update that file as well?

https://github.com/wso2/performance-is/blob/master/common/summary/summary-modifier.py#L31-L47

Reply:
Added 40caa9f3c1bd8a7e58324e657a58265fc0fe7bbb